tests/demo/demo.cpp

```diff
@@ -23,12 +23,13 @@
 #include "scene_viknell.hpp"
 #include "scene_emibl.hpp"
 #include "scene_sponza.hpp"
+#include "scene_shadows_transparency.hpp"
 
 #include "model_loader_assimp.hpp"
 #include "model_loader_tinygltf.hpp"
 #include "d3d12/d3d12_dynamic_descriptor_heap.hpp"
 
-using DefaultScene = ViknellScene;
+using DefaultScene = TransparencyScene;
```

Comment:
Is this the new default scene for master?

Reply:
Leftout code